modules/ROOT/pages/pages-development.adoc

```diff
@@ -1,14 +1,14 @@
 = Pages development
-
 :description: A page presents information to a user or can also collect information entered by a user. +
 
-A page presents information to a user or can also collect information entered by a user. +
+
+{description} +
 Displayed in a browser, a xref:pages.adoc[page] is intended for use in applications.
 
 == Provided pages
 
 You do not need to create all your pages from scratch. Bonita already provides some pages. +
-Bonita Portal "Resources" section lists all the non-UI Designer-made pages, that can be reused as is in an application.
+Bonita Administrator Application "Resources" tab lists all the non-UI Designer-made pages, that can be reused as is in an application.
```

Comment:
hat can be reused as is in an application. => hat can be reused "as-is" in an application.

Reply:
Added